src/osd/OSD.cc

```diff
@@ -1757,8 +1757,13 @@ void OSDService::queue_scrub_event_msg(PG* pg,
   dout(15) << "queue a scrub event (" << *msg << ") for " << *pg
            << ". Epoch: " << epoch << " token: " << act_token << dendl;
 
+  int64_t cost_for_queue = cct->_conf->osd_scrub_cost;
```

Comment:
Factor this out to a new OSDService method for e.g., `OSDService::get_scrub_cost()` that will work with both wpq and mclock. That way, this method can be re-used with the longer term solution too.

Reply:
+1